src/app/pages/forms/components/inputs/components/ratinginputs/ratinginputs.component.ts

```diff
@@ -0,0 +1,20 @@
+import { RatingComponent } from 'ng2-bootstrap/ng2-bootstrap';
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'rating-inputs',
+  directives: [RatingComponent],
+  template: require('./ratinginputs.html')
+})
+
+export class Rating {
+  private rate1:number = 3;
```

Comment:
@Tibing please make the private members start with _ symbol as per the other code style (rate1 should be _rate1). Or, make them publicly accessible.